### ISSUE TITLE
Several match_fields

### ISSUE THREAD

**Issue by bedilbek** [fopina/django-bulk-update-or-create#5]
I guess, for now, the library does not support several match_fields as the following from native update_or_create:
```python
    SomeModel.objects.update_or_create(name='something', status='something', defaults={'description': 'something'})
```

**Comment by bedilbek** [comment]
Yeah, I have just seen it in [ToDo](https://github.com/fopina/django-bulk-update-or-create#todo)

**Comment by fopina** [comment]
fixed by #6 :)
